content/en/blog/2021/life-without-crds/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Life Without CRDs"
```

Comment:
I would call this something like "Zero Config Istio"

The CRDs are all still there :)

Reply:
I was actually planning to not install the CRDs, but unfortunately telemetry would break so its a bit awkward. I think we could actually not setup the CRDs, and exclude envoyfilter/wait for Telemetry API to land, or change the title. 

I think changing the title is probably for the best, unless people actually think CRD == complex, rather than using CRDs as a proxy for complexity?

Reply:
Yeah I think people use CRDs as a proxy for complexity, so changing the title is good. If we don't install the CRDs then there is friction to go from zero-config to first-config, and we don't want that IMO. It is really about what they are forced to learn about in order to get their job done.

Reply:
+1 changing makes sense.  :) The key is no change to their code and apply Istio resources as needed incrementally.